README.md

```diff
@@ -0,0 +1,219 @@
+* encourage modularization for a clearer separation of concerns,
+* enforce dependency version consistency across different local packages,
+* improve the developer experience and reduces the build file size by ensuring there is only one version of a dependency across all packages in the repository.
+
+[//]: # (To learn more about *why exactly* you should consider using binge, read our [announcement blog post]https://tech.signavio.com/2017/package-management-binge.)
+
+## Getting started with binge
+Install binge by running ``yarn global add binge``.
+
+**Note**: binge requires [Yarn](https://yarnpkg.com/lang/en/) and doesn't support npm.
+
+### Repository structure
+Structure your repository as follows:
+
+```
+…
+- client
+    - package1
+        - …
+        - src
+        - .babelrc
+        - .npmignore
+        - package.json
+        - yarn.lock
+    - …
+    - packageN
+        - …
+    - app
+        - src
+        - package.json
+        - …    
+…
+```
+
+Your client folder needs to contain a root-level folder for each local package, plus one additional folder that contains the global entry point to (the *root* of) your app.
+In our example (see: [./examples/watch-app](./examples/watch-app)), we name this folder ``app``.
+
+**Note:** binge supports nested local packages. However, we recommend you don't nest your packages.
+This reduces complexity and allow a clearer overview.
+
+Local packages have the same structure as ordinary npm packages, except that they are **not versioned**. 
+
+The ``package.json`` in the root folder lists the local packages as dependencies:
+
+```json
+{
+  "name": "app",
+  "dependencies": {
+      "i-filter-stuff": "file:../i-filter-stuff",
+      "i-provide-text": "file:../i-provide-text"
+  },
+  …
+}
+```
+
+To build the app, navigate to the ``app`` directory and run ``binge bootstrap``.
+binge now installs the dependencies in the local packages, builds the local packages and deploys them and their dependencies to the ``node_modules`` folder of the ``app`` directory. At that, binge moves shared dependencies up the dependency tree to avoid unnecessary duplication of code.
+
+To run the app and watch the source files, run ``binge watch`` in the ``app`` directory.
+
+## Reference documentation
+This section lists and explains all binge commands.
+You find minimal example apps that illustrate the commands at [./examples](./examples).
+
+* ``bootstrap``:
+
+    Installs, builds and deploys the local package tree.
+    For example, running ``binge bootstrap`` in [./examples/bootstrap/root](./examples/bootstrap/root):
+    * installs the dependencies of the three local packages ``i-filter-stuff``, ``i-print-stuff`` and ``i-prodivde-colors``,
+    * builds the three local packages,
+    * deploys the local packages and their dependencies to the ``node_modules`` folder of the ``root`` directory. At that, binge moves shared dependencies up the dependency tree to avoid unnecessary duplication of code.
```

Comment:
Remove the last sentence - (At that, binge moves shared dependencies up the dependency tree to avoid unnecessary duplication of code.)

We could also add that:
That not only the app gets its node_modules bootstraped, but also all local-packages get bootstraped.